src/lib.js

```diff
@@ -165,6 +165,7 @@ export default class Migrator {
         this.log('There are no migrations to run'.yellow);
         this.log(`Current Migrations' Statuses: `);
         await this.list();
+        return []
```

Comment:
Ah I see what's going on. I was referring to another bug I noticed lol where in API mode (as npm package) it was throwing an error as well, when it shouldn't in either case.

Thanks! @river-jade  I won't merge this so I can fix both issues~! <3